src/libostree/ostree-repo-pull.c

```diff
@@ -3592,10 +3606,12 @@ ostree_repo_pull_with_options (OstreeRepo             *self,
       (void) g_variant_lookup (options, "subdir", "&s", &dir_to_pull);
       (void) g_variant_lookup (options, "subdirs", "^a&s", &dirs_to_pull);
       (void) g_variant_lookup (options, "override-remote-name", "s", &pull_data->remote_refspec_name);
+#if defined(HAVE_GPGME)
       opt_gpg_verify_set =
         g_variant_lookup (options, "gpg-verify", "b", &pull_data->gpg_verify);
```

Comment:
Maybe we should explicitly error out if this is set to `TRUE` but it's disabled at build time?  Dunno.

Reply:
added the appropriate check for that.